src/Scaler/Services/CosmosDbMetricProvider.cs

```diff
@@ -3,47 +3,73 @@
 using System.Net;
 using System.Net.Http;
 using System.Threading.Tasks;
+using Azure.Identity;
+using Microsoft.AspNetCore.Http;
 using Microsoft.Azure.Cosmos;
 using Microsoft.Extensions.Logging;
-
+using Microsoft.Identity.Client.Platforms.Features.DesktopOs.Kerberos;
+using Container = Microsoft.Azure.Cosmos.Container;
+using System.Diagnostics.Metrics;
+using OpenTelemetry;
+using OpenTelemetry.Metrics;
+using Microsoft.Identity.Client;
 namespace Keda.CosmosDb.Scaler
 {
     internal sealed class CosmosDbMetricProvider : ICosmosDbMetricProvider
     {
         private readonly CosmosDbFactory _factory;
         private readonly ILogger<CosmosDbMetricProvider> _logger;
 
+		static Meter s_meter = new Meter("OrderProcessor.CFStore", "1.0.0");
```

Comment:
Set visibility of properties explicitly. Use standard C# property names and put newlines between property declarations and methods.

Reply:
ok